client/app/bundles/shared/components/Tag.jsx

```diff
@@ -0,0 +1,26 @@
+//@flow
+import React from "react";
+import css from "./Tag.scss";
+
+type Props = {
+  dark?: boolean,
+  normal?: boolean,
+  label?: string,
+  id?: string,
+  };
+
+export default class Tag extends React.Component<Props> {
+  constructor(props: Props)
+  {super(props);}
+  render()
+   {
+     const {dark,normal,label,id} = this.props;
+     const labelClassNames = `${css.label} ${dark ? css.dark : ""}${normal ? css.normal : ""}`;
+
+          return (
+        <span>
+          <p className = {labelClassNames}>{label}</p>
```

Comment:
I still don't think the use of `<p>` is contextually correct in this component, but that's a minor thing! I'm of the belief that `<p>` should be exclusively used for copy blocks. Not a game stopper, just my own opinion!

Reply:
Thanks for the insight ! I did not think of it in this way, shall change it to div. :D